src/ShopifyApp/Services/ApiHelper.php

```diff
@@ -470,4 +470,29 @@ protected function doRequestGraphQL(string $query, array $payload = null)
 
         return $response;
     }
+
+    /**
+     * @param  Session  $session
+     * @return mixed
+     */
+    private function getShop(Session $session = null)
```

Comment:
Lets have this function return `ShopDomain` value to ensure the shop domain is formatted correctly.
Then line 45, we can do `->toNative()` to have it back into a string for use with configs.